editor/editor_inspector.cpp

```diff
@@ -1787,6 +1836,50 @@ void EditorInspector::update_tree() {
 		_parse_added_editors(main_vbox, ped);
 	}
 
+	// Disable properties managed by a parent container
+	{
```

Comment:
This is really dirty. We should simply need a list of the properties to disable and disable the corresponding EditorProperty.
That probably requires adding a function in EditorInspectorPlugin that allows retrieving the EditorProperty corresponding to a given property name.

Reply:
Understood. I'll rework the PR to meet the required changes.